cost/azure/reserved_instances/mca_recommendations/azure_reserved_instance_recommendations.pt

```diff
@@ -26,6 +26,24 @@ parameter "param_days" do
   type "string"
 end
 
+parameter "param_type" do
+  category "RI"
+  label "Resource Type"
+  default "VirtualMachines"
+  description "The resource type used for reservation recommendations"
+  allowed_values "VirtualMachines","SQLDatabases","PostgreSQL","ManagedDisk", "MySQL", "RedHat", "MariaDB", "RedisCache", "CosmosDB", "SqlDataWarehouse", "SUSELinux", "AppService", "BlockBlob", "AzureDataExplorer", "VMwareCloudSimple"
+  type "string"
```

Comment:
Given my experience of applying a similar policy for AWS, I'm wondering if it might be better for this parameter to be an `array` so that a single apply can cover multiple resource types rather than requiring a single apply for each type.